tempesta_fw/str.c

```diff
@@ -119,6 +119,23 @@ tfw_str_add_duplicate(TfwPool *pool, TfwStr *str)
 	return dup_str;
 }
 
+/**
+ * Compute the whole string's length, including all the EOL's.
+ */
+unsigned long
+tfw_strlen(const TfwStr *str)
```

Comment:
Yeah, @keshonok made reasonable note about name of the function. Basically, TfwStr has `len` member to get the string length immediately instead of computation via `strlen()` or similar functions, so I thing `strlen`-containg functions will be misleading for TfwStr. I'd name the function like `tfw_str_dup_len()`.

Meantime, I'm also always confused by `tfw_str_total_len()` - knowing TfwStr internals I expect that the function returns length of all the chunks of duplicates. So I'd rename the function to `tfw_str_eol_len()` since `total` doesn't say anything what `total` is: EOL, chunks or duplicates.